src/main/kotlin/com/github/imflog/schema/registry/tasks/register/RegisterTaskAction.kt

```diff
@@ -27,9 +29,9 @@ class RegisterTaskAction(
     fun run(): Int {
         var errorCount = 0
         writeOutputFileHeader()
-        subjects.forEach { (subject, path, type, references: List<SchemaReference>, localReferences) ->
+        subjects.forEach { (subject, path, type, references: List<SchemaReference>, localReferences, metaData, ruleSet, normalize) ->
```

Comment:
We start to have too many things de-structured. We probably should go back to use a variable and use accessors.